src/main/java/edu/bu/metcs673/trackr/bankaccount/BankAccountService.java

```diff
@@ -16,50 +16,56 @@
 @Component
 public interface BankAccountService {
 
-    /**
-     * Creates a new bank account using the provided object, associate to the user
-     * making the request. Fields not provided by the user will be filled in with
-     * their default values.
-     *
-     * @param bankAccountInput
-     * @return BankAccount
-     */
-    public BankAccount createBankAccount(BankAccountDTO bankAccountInput, TrackrUser user);
+	/**
+	 * Creates a new bank account using the provided object, associate to the user
+	 * making the request. Fields not provided by the user will be filled in with
+	 * their default values.
+	 *
+	 * @param bankAccountInput
+	 * @return BankAccount
+	 */
+	public BankAccount createBankAccount(BankAccountDTO bankAccountInput, TrackrUser user);
 
-    /**
-     * Modifies a BankAccount record with the provided id, if the provide TrackrUser
-     * is the same one who created the record. Value in the BankAccountDTO will
-     * overwrite existing values.
-     *
-     * @param bankAccountInput
-     * @param user
-     * @param id
-     * @return BankAccount
-     */
-    public BankAccount modifyBankAccount(BankAccountDTO bankAccountInput, TrackrUser user, long id);
+	/**
+	 * Modifies a BankAccount record with the provided id, if the provide TrackrUser
+	 * is the same one who created the record. Value in the BankAccountDTO will
+	 * overwrite existing values.
+	 *
+	 * @param bankAccountInput
+	 * @param user
+	 * @param id
+	 * @return BankAccount
+	 */
+	public BankAccount modifyBankAccount(BankAccountDTO bankAccountInput, long userId, long id);
 
-    /**
-     * Updates a status in a BankAccount record with the provided id, if the
-     * provided TrackrUser is the same associated to the record. Sets
-     * 'accountStatus' to INACTIVE which prevents new transactions from being
-     * associated to the BankAccount record.
-     *
-     * @param user
-     * @param id
-     * @return
-     */
-    public void deactivateBankAccount(TrackrUser user, long id);
+	/**
+	 * Updates a status in a BankAccount record with the provided id, if the
+	 * provided TrackrUser is the same associated to the record. Sets
+	 * 'accountStatus' to INACTIVE which prevents new transactions from being
+	 * associated to the BankAccount record.
+	 *
+	 * @param user
```

Comment:
Actual parameter name is `userId`. There are a lot of issues like this in this class.